edge/pkg/metamanager/dao/upgradedb/confirm_db.go

```diff
@@ -0,0 +1,143 @@
+package upgradedb
```

Comment:
As we have discussed before, we decided save confirm data in metav2 but not create a new table, right?

Reply:
I created a table called `NodeUpgradeConfirmTable` and registered this table using `orm.RegisterModel`,and then I use database to save `NodeTaskRequest` and `NodeUpgradeJobRequest`.